locales/locales.php

```diff
@@ -1998,6 +1998,18 @@ public function __construct() {
 		$pcd->nplurals = 2;
 		$pcd->plural_expression = 'n > 1';
 
+		$pa_pk = new GP_Locale();
+		$pa_pk->english_name = 'Punjabi (Pakistan)';
+		$pa_pk->native_name = 'پنجابی';
+		$pa_pk->lang_code_iso_639_1 = 'pa';
+		$pa_pk->lang_code_iso_639_2 = 'pnb';
+		$pa_pk->country_code = 'pk';
+		$pa_pk->wp_locale = 'pa_PK';
+		$pa_pk->slug = 'pa-pk';
+		$pa_pk->nplurals = 2;
+		$pa_pk->plural_expression = 'n != 1';
```

Comment:
Is this really different from Panjabi in India? We should probably use the same to align with the CLDR data.

This would also fix the current test failure: https://travis-ci.org/github/GlotPress/GlotPress-WP/jobs/704585915#L348-L357

Reply:
Yes, that seems to be the case - I shared this change with the requester, and will wait for his response.
https://make.wordpress.org/polyglots/2019/05/29/hi-i-want-help-wordpress-with-punjabi-translation-2/#comment-288001

Reply:
I didn't receive a reply from the requestor but my colleague who is a Punjabi (India) native speaker confirmed the plural format is the same for both locales.